Guided Investigation - MDATP Webshell Alerts.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<p>This isn't a must do but you can use md to display headings as well with md("Heading", "bold")</p><p> </p><p>Also you use q as the variable name for multiple queries, its good practice to avoid this if you can - maybe use something like aspx_q.</p><p> </p><p>You check aspx_data.empty but if the querie fails due to timeout or some other error it will return None, so its good to add an isinstance catch as well:</p><p>if isinstance(aspx_data, pd.DataFrame) and not aspx_data.empty:</p><p> </p><p>Also for warning messages we have md_warn that you can choose to use if you like.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Thanks for these, I've adopted all proposed changes.</p>

Reply:
<p>You should also be able to send md() HTML - it is doing display(HTML(txt)) under the covers but it does mess with styles a bit so might screw up a lot of your beautiful HTML. :-)</p>

Reply:
<p>haha, you're very kind calling it beautiful Ian - I'll adopt the use of md() in future.</p>